src/components/Header.js

```diff
@@ -42,11 +42,11 @@ class Header extends React.Component {
   }
 
   // Filters what gets displayed based on the user inputs
-  /*
-  fetchCurRecipes = async () => {
-    await fetch(`/recipes?page=${this.state.currentPage}&limit=${this.state.pageSize}&` +
-     `${this.createIngredientQueryParam()}name=${this.state.name}&min=${this.state.minTime}` +
-     `&max=${this.state.maxTime}&username=${this.state.createdBy}&recipes=${LZString.decompress(localStorage.getItem("recipes"))}`, {
+  fetchRecipes = () => {
+    fetch(`/recipes?page=${this.state.currentPage}&limit=` +
```

Comment:
+4 here instead of +2

Reply:
According to Google Styles for JS, this should fall under block indentation so only +2. The below lines would be under line wrapping so +4 from first line until the next JSON block.